doc/source/whatsnew/v0.21.0.txt

```diff
@@ -116,6 +116,8 @@ Other API Changes
 Deprecations
 ~~~~~~~~~~~~
 - :func:`read_excel()` has deprecated ``sheetname`` in favor of ``sheet_name`` for consistency with ``.to_excel()`` (:issue:`10559`).
+- :func:`Series.argmax` has been deprecated in favor of :func:`Series.idxmax` (:issue:`16830`)
```

Comment:
Also the docstrings of `argmax`/`argmin` will need this description (that they will later return the positional index)